3529/2/Lib/Utiles/image_loader.rb

```diff
@@ -0,0 +1,7 @@
+module ImageLoader
+  def load_image
+    file_id =  payload['photo'].first['file_id']
+    response = HTTParty.get("https://api.telegram.org/bot#{TOKEN}/getFile?file_id=#{file_id}")
+    "https://api.telegram.org/file/bot#{TOKEN}/#{response['result']['file_path']}"
```

Comment:
All long strings to constant please

Reply:
I can't find a way how to do it in such situation. 'cause i need to pass hound test, and it requires freeze constants, but this constants should be changed depending on images... :(
So i don't change it in last commit

Reply:
`"https://api.telegram.org/bot#{TOKEN}/getFile?file_id=` this part will never be changed